src/applications/disability-benefits/view-payments/components/responsive-table/ResponsiveTable.jsx

```diff
@@ -45,24 +46,29 @@ class ResponsiveTable extends Component {
     );
   };
 
+  capitalizeFirstLetter = string => {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+  };
+
   renderRow = item => {
     const { fields } = this.props;
     let extraClass = '';
     return (
-      <tr key={item.id} className={borderClasses}>
+      <tr key={item.id} className={`${borderClasses} ${rowPaddingClass}`}>
         {fields.map((field, index) => {
           // This is to right align the amount field and account number fields
           // since they are numeric
           if (index === 1 || index === 5) {
-            extraClass = 'vads-u-text-align--right';
+            extraClass =
+              'vads-u-text-align--left medium-screen:vads-u-text-align--right';
           } else {
             extraClass = '';
           }
           return (
             <td
               data-index={index}
               className={`${borderClasses} ${extraClass}`}
-              data-label={field.value}
+              data-label={`${this.capitalizeFirstLetter(field.value)}:`}
```

Comment:
Curious, what's the `:` is for at the end of the data-label??

Reply:
its there because they want the mobile design to show data like this `date: June 2, 2020` so they want to colon in there but we don't want to modify the fields data structure and add that because there are other things that rely on that data structure.